tests/terraform/parser/resources/parser_scenarios/ternary_793/main.tf

```diff
@@ -0,0 +1,18 @@
+variable "metadata_http_tokens_required" {
+  type        = bool
+  default     = true
+  description = "Whether or not the metadata service requires session tokens"
+}
+
+resource "aws_instance" "foo" {
+  ami           = "ami-005e54dee72cc1d00" # us-west-2
+  instance_type = "t2.micro"
+
+  root_block_device {
+    encrypted = true
+  }
+
+  metadata_options {
+    http_tokens = (var.metadata_http_tokens_required) ? "required" : "optional"
```

Comment:
@robeden Can we test multi-line ternary? i.e.:
```terraform
 http_tokens = (var.metadata_http_tokens_required) ? 
    "required" : "optional"
```

Reply:
Looks like the HCL2 parser dies on that:

> Unexpected token Token(__ANON_0, '\\n') at line 10, column 35.
> Expected one of: 
>  * LSQB
>  * LBRACE
>  * LPAR
>  * NULL
>  * __ANON_11
>  * DECIMAL
>  * FALSE
>  * __ANON_3
>  * __ANON_10
>  * STRING_LIT
>  * MINUS
>  * TRUE
>  * BANG

Reply:
I added this to the "ternaries" test case:
```
  local_true = true
  multiline = (local.local_true) ?
    "correct" : "wrong"
```

(Note: local evals inside locals isn't valid in terraform, but our parser would handle it... if the HCL2 parser would.)

Reply:
@robeden Is it on a commit you haven't pushed yet?

Reply:
Yes, sorry. Pushed now.